tpm2/structures.go

```diff
@@ -295,8 +289,7 @@ func (p *ECCParams) encode() ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("encoding KDF: %v", err)
 	}
-	x, y := p.Point.x().Bytes(), p.Point.y().Bytes()
-	point, err := tpmutil.Pack(tpmutil.U16Bytes(x), tpmutil.U16Bytes(y))
+	point, err := tpmutil.Pack(tpmutil.U16Bytes(p.Point.XRaw), tpmutil.U16Bytes(p.Point.YRaw))
```

Comment:
`tpmutil.U16Bytes` isn't needed here (as that's already the type of `p.Point.XRaw`.

`tpmutil.Pack(p.Point.XRaw, p.Point.YRaw)` should work.

Reply:
good catch, done